### PR TITLE
docs(blog): ibis, duckdb geo and lonboard for overture maps

### PR DESCRIPTION
**Pull request by ncclementi**
Closes https://github.com/ibis-project/ibis/issues/9598

Includes changes after duckdb 1.1.1 release and review comments. 

cc: @kylebarron

### CODE REVIEW

File: docs/posts/ibis-overturemaps/index.qmd [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+directly from an `Ibis` table, adding more simplicity and speed to your
+geospatial analysis.
+
+Let’s dive into how these tools come together.
+
+## Installation
+
+First make sure you have `duckdb=1.1.1`, then install Ibis with the dependencies
+needed to work with geospatial data using DuckDB. To be able to read GeoParquet
+files the DuckDB version should be `>=1.1.1`.
```
Comment (kylebarron) [inline docs/posts/ibis-overturemaps/index.qmd:29]: You mention the duckdb version constraint twice here

File: docs/posts/ibis-overturemaps/index.qmd [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+```bash
+$ pip install duckdb==1.1.1
```
Comment (kylebarron) [inline docs/posts/ibis-overturemaps/index.qmd:32]: I'd suggest that we suggest >=1.1.1 for users

File: docs/posts/ibis-overturemaps/index.qmd [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+First we import Ibis, its [deferred expression object](https://ibis-project.org/reference/expression-generic.html#ibis.expr.api.deferred) `_` ,
+and we use our default backend, DuckDB:
+```{python}
+import ibis
+from ibis import _
+
+con = ibis.get_backend() # default duckdb backend
+```
+
+With Ibis and DuckDB we be more specific about the data we want thanks to the
```
Comment (kylebarron) [inline docs/posts/ibis-overturemaps/index.qmd:60]: `we can be`
?

**Review by kylebarron — APPROVED** [review]

**Review by kylebarron — APPROVED** [review]

**Review by lostmygithubaccount — APPROVED** [review]

File: docs/posts/ibis-overturemaps/index.qmd [hunk 4]
```diff
@@ -0,0 +1,217 @@
+---
+title: "From query to plot: Exploring GeoParquet Overture Maps with Ibis, DuckDB, and Lonboard"
+author: Naty Clementi and Kyle Barron
+date: 2024-09-13
```
Comment (lostmygithubaccount) [inline docs/posts/ibis-overturemaps/index.qmd:4]: want to update the date but could be done in a follow up :)

**Review by cpcloud — APPROVED** [review]

**Pull request merged**